runner/utils/driver/docker/helpers.go

```diff
@@ -62,3 +63,22 @@ func (d *docker) pullImageIfDoesntExists(ctx context.Context, projectId string,
 	}
 	return nil
 }
+
+func getSpaceCloudHostsFilePath() string {
```

Comment:
You cannot use these functions here since these will be invoked inside the runner and not on the host

Reply:
Instead accept the `HOSTS_FILE_PATH` environment variable like we are doing for secrets in the `NewDockerDriver` method. Don't forget to add the env variable in space cli setup